src/AppBundle/Resources/views/Macros/macros.html.twig

```diff
@@ -87,72 +87,77 @@
-        {'section':'reportOverview', 'transKey': reportOverviewTransKey, 'link': path('odr_overview', {'odrId': report.id})},
+            {'section':'reportOverview', 'link': path('odr_overview', routeParams)},
+            {'section':'visitsCare','link': path('odr_visits_care', routeParams)},
+            {'section':'deputyExpenses','link': path('odr_deputy_expenses', routeParams)},
+            {'section':'incomeBenefits','link': path('odr_income_benefits', routeParams)},
+            {'section':'bankAccounts','link': path('odr_bank_accounts', routeParams)},
+            {'section':'assets','link': path('odr_assets', routeParams)},
+            {'section':'debts','link': path('odr_debts', routeParams)},
+            {'section':'actions', 'link': path('odr_actions', routeParams)},
+            {'section':'otherInfo', 'link': path('odr_other_info', routeParams)},
+            {'section':'reportOverview', 'transKey': reportOverviewTransKey, 'link': path('odr_overview', routeParams)},
         ] %}
+    {% else %}
+
+        {# when all merged and stable, remove the section option and use the key #}
+        {% set routeParams = {'reportId': report.id} %}
+
+        {# all the possible section for all the reports (A-Z order) #}
+        {% set allSectionsAvailable = {
+            'actions': {'section':'actions', 'link': path('actions', routeParams)},
+            'assets':  {'section':'assets', 'link': path('assets', routeParams)},
+            'bankAccounts': {'section':'bankAccounts','link': path('bank_accounts', routeParams)},
+            'contacts': {'section':'contacts', 'link': path('contacts', routeParams)},
+            'debts': {'section':'debts', 'link': path('debts', routeParams)},
+            'deputyExpenses': {'section': 'deputyExpenses','link': path('deputy_expenses', routeParams)},
+            'decisions': {'section':'decisions', 'link': path('decisions', routeParams)},
+            'documents': {'section':'documents', 'link': path('documents', routeParams)},
+            'gifts': {'section':'gifts','link': path('gifts', routeParams)},
+            'lifestyle': {'section':'lifestyle','link': path('lifestyle', routeParams)},
+            'moneyTransfers': {'section':'moneyTransfers', 'link': path('money_transfers', routeParams)},
+            'moneyIn': {'section':'moneyIn', 'link': path('money_in', routeParams)},
+            'moneyOut': {'section':'moneyOut', 'link': path('money_out', routeParams)},
+            'moneyInShort': {'section':'moneyInShort', 'link': path('money_in_short', routeParams)},
+            'moneyOutShort': {'section':'moneyOutShort', 'link': path('money_out_short', routeParams)},
+            'otherInfo': {'section':'otherInfo', 'link': path('other_info', routeParams)},
+            'paDeputyExpenses': {'section': 'paFeeExpense','link': path('pa_fee_expense', routeParams)},
+            'visitsCare': {'section':'visitsCare','link': path('visits_care', routeParams)},
+        } %}
+
```

Comment:
Eventually you could have the 6 default sections in config and just append to the array for each report type